includes/html/api_functions.inc.php

```diff
@@ -2264,12 +2268,20 @@ function list_logs(Illuminate\Http\Request $request, Router $router)
     }
 
     if ($from) {
-        $query .= " AND $timestamp >= ?";
+        if (is_numeric($from)) {
+            $query .= " AND $id_field >= ?";
+        } else {
+            $query .= " AND $timestamp >= ?";
+        }
         $param[] = $from;
     }
 
     if ($to) {
-        $query .= " AND $timestamp <= ?";
+        if (is_numeric($to)) {
```

Comment:
What if to or from is a timestamp?  How can you tell the difference?

Reply:
I am not aware you can actually pass numerical timestamps ? As the database fields are datetime (I always used dates as string as parameter in this case)

Reply:
Tested locally and currently it looks like numeric timestamps are ignored.